OrbitService/ProcessServiceImpl.cpp

```diff
@@ -69,6 +69,23 @@ Status ProcessServiceImpl::GetModuleList(ServerContext* /*context*/,
   return Status::OK;
 }
 
+Status ProcessServiceImpl::GetTracepointList(
+    grpc::ServerContext*, const orbit_grpc_protos::GetTracepointListRequest*,
+    orbit_grpc_protos::GetTracepointListResponse* response) {
+  LOG("Sending tracepoints");
+
+  const auto tracepoint_infos = LinuxUtils::ListTracepoints();
+  if (!tracepoint_infos) {
+    return Status(StatusCode::NOT_FOUND, tracepoint_infos.error().message());
+  }
+
+  for (const auto& tracepoint_info : tracepoint_infos.value()) {
+    *(response->add_tracepoints()) = tracepoint_info;
+  }
```

Comment:
Instead of this for loop:
*response->mutable_tracepoints() = {tracepoint_infos.begin(), tracepoin_tinfos.end()};